CONTRIBUTING.md

```diff
@@ -43,7 +43,7 @@ You *must* include a `changeset` file in your PR when making a change that would
 Adding a `changeset` file is easy:
 
 1. Navigate to the root of the monorepo.
-2. Run `yarn changeset`. You'll be prompted to select packages to include in the changeset. Use the arrow keys to move the cursor up and down, hit the `spacebar` to select a package, and hit `enter` to confirm your selection. Select *all* packages that require a new release as a result of your PR.
+2. Run `pnpm changeset`. You'll be prompted to select packages to include in the changeset. Use the arrow keys to move the cursor up and down, hit the `spacebar` to select a package, and hit `enter` to confirm your selection. Select *all* packages that require a new release as a result of your PR.
```

Comment:
Unrelated to this pr but this is too much detail.  Once you run pnpm changeset you can figure out the rest because the prompt guides you along